pkgs/state-transition/src/lib.zig

```diff
@@ -56,85 +57,36 @@ test "apply transition on mocked chain" {
-    try std.testing.expect(std.mem.eql(u8, &test_genesis_block_root, &expected_genesis_block_root));
-
-    // check produced block 1 state root
-    var expected_block1_state_root: [32]u8 = undefined;
-    _ = try std.fmt.hexToBytes(expected_block1_state_root[0..], "993a459ace700aaeacf2fd3ec20b0f9efd3acae93ff904c5adbc1997b025ea97");
-    try std.testing.expect(std.mem.eql(u8, &expected_block1_state_root, &mock_chain.blocks[1].message.state_root));
-
-    // 7. check block 1 root
-    var block1_root: [32]u8 = undefined;
-    try ssz.hashTreeRoot(types.BeamBlock, mock_chain.blocks[1].message, &block1_root, std.testing.allocator);
-    var expected_block1_root: [32]u8 = undefined;
-    _ = try std.fmt.hexToBytes(expected_block1_root[0..], "4851312e74b73c99581772286a853ee8fc6a9d2fde00c2bd21e9c7b966a6c238");
-    try std.testing.expect(std.mem.eql(u8, &block1_root, &expected_block1_root));
-
-    // 9. run and check state transition
-    // TODO: the previous process block should have been run on cloned state so we have the original pre
-    // state here to run the state transition. for now regen same genesis state
-    var state = try utils.genGenesisState(std.testing.allocator, test_config);
-    try apply_transition(std.testing.allocator, &state, mock_chain.blocks[1], .{});
-    var post_state_root: [32]u8 = undefined;
-    try ssz.hashTreeRoot(types.BeamState, state, &post_state_root, std.testing.allocator);
-
-    try std.testing.expect(std.mem.eql(u8, &post_state_root, &expected_block1_state_root));
-}
-
-test "genStateBlockHeader" {
-    // 1. setup genesis config
-    const test_config = types.GenesisSpec{
-        .genesis_time = 1234,
-    };
-
-    var arena_allocator = std.heap.ArenaAllocator.init(std.testing.allocator);
-    defer arena_allocator.deinit();
-    const allocator = arena_allocator.allocator();
-
-    const mock_chain = try genMockChain(allocator, 2, test_config);
-
-    var beam_state = mock_chain.genesis_state;
-    for (0..mock_chain.blocks.len) |i| {
-        // get applied block
-        const applied_block = mock_chain.blocks[i];
-        var applied_block_root: types.Root = undefined;
-        try ssz.hashTreeRoot(types.BeamBlock, applied_block.message, &applied_block_root, allocator);
-
-        const state_block_header = try utils.genStateBlockHeader(allocator, beam_state);
-        var state_block_header_root: types.Root = undefined;
-        try ssz.hashTreeRoot(types.BeamBlockHeader, state_block_header, &state_block_header_root, allocator);
-
-        try std.testing.expect(std.mem.eql(u8, &applied_block_root, &state_block_header_root));
-
-        if (i < mock_chain.blocks.len - 1) {
-            // apply the next block
-            const block = mock_chain.blocks[i + 1];
-            try apply_transition(allocator, &beam_state, block, .{});
-        }
-    }
-}
+// test "genStateBlockHeader" {
```

Comment:
same here: I don't recommend disabling tests

Reply:
yes, uncommenting this